pkg/policy/existing.go

```diff
@@ -27,7 +27,7 @@ func (pc *PolicyController) processExistingResources(policy kyverno.PolicyInterf
 	pc.rm.Drop()
 
 	for _, rule := range autogen.ComputeRules(policy) {
-		if !rule.HasValidate() && !rule.HasVerifyImages() {
+		if !rule.HasValidate() && !rule.HasVerifyImages() || !rule.HasGenerate() {
```

Comment:
Why do you need the policy controller to reconcile generate policies? Policy controller is used to create policy reports.

Reply:
yes , have to clean this